how-to/business-logic/how-to-determine-the-user-account.md

```diff
@@ -0,0 +1,56 @@
+---
+title: How to Determine the User Account Which Is Making the Request
+description: An example to perform a check and use the current user account properties

```

Comment:
My comment for the previous article is still applicable. Consider adding a mention of the service/API in the description.